README.md

```diff
@@ -1,7 +1,7 @@
 # NumpyNet
 ![Build Status](https://api.travis-ci.org/UptakeOpenSource/numpynet.svg?branch=master)
 
-![alt text](https://mybroadband.co.za/news/wp-content/uploads/2017/04/Neural-network.jpg)
+![](https://raw.githubusercontent.com/mairuoliu/numpynet/master/readme_figures/demo.gif)
```

Comment:
@mairuoliu , this is a sweet GIF!  I have a few minor requests.    Could you: 

1. Move this gif below the "what is NumpyNet" section (line 10)
2. Reduce the file size of this GIF.  It is currently 12 MB, and I think it can be around 4 MB
3. While I think you for linking to your fork so we can see the gif, please ensure to modify the URL to uptake.